partials/footer.hbs

```diff
@@ -2,29 +2,28 @@
     <div class="social">
         {{#if @site.facebook}}
```

Comment:
Please remove this `if` block since we're using our own list of social media links.